app/models/call.rb

```diff
@@ -18,15 +19,22 @@ class Call
                       inclusion: { in: allowed_statuses }
   validates :created_by, presence: true
 
+<<<<<<< HEAD
```

Comment:
Can you remove these merge conflict artifacts? Go ahead and delete lines 22-34, and the one on line 39 too, if you would be so kind!